src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1864,9 +1867,11 @@ internal bool CreateDebugDocuments(DebugDocumentsBuilder documentsBuilder, IEnum
                 return CompileMethods(
                     moduleBuilder,
                     emittingPdb,
-                    diagnostics,
-                    filterOpt,
-                    cancellationToken);
+                    emitMetadataOnly: false,
+                    emitTestCoverageData: false,
```

Comment:
Might be helpful to put that in a comment, please.

---
In reply to: [109278314](https://github.com/dotnet/roslyn/pull/18339#discussion_r109278314) [](ancestors = 109278314)